casks/tunnel-client-app.rb

```diff
@@ -0,0 +1,10 @@
+cask 'tunnel-client-app' do
+  version '1.0.0'
+  sha256 'bc8dcb14d803e098facb1373e37123bcd74a24cc2667167c7953a09f3819e985'
+
+  url 'https://storage.googleapis.com/tunnel-client-app-binaries/TunnelClient.app.tar.gz'
```

Comment:
Does shipit put this file up there? Also, can we add a version folder? i.e. .../1.0.0/TunnelClient.app.tar.gz` so it's easily identifiable later and we can retain old versions if needed?

Reply:
> Does shipit put this file up there?

Not right now, this one was "manual", I added a script (that's executed manually) to build & package & push it to GCS. Working on CI/CD for it next tho :) I mostly wanted feedback early.

> Also, can we add a version folder?

Yeah, good idea to keep it clean 👍 can do this for SHA of commits too.
Any thoughts/preferences on using folders or different filenames of the archive for namespacing?
@BlakeMesdag @johnjmartin @andrewryder @alanctgardner 
e.g. 
1. `/1.0.0/TunnelClient.app.tar.gz` and `/somecommitsha/TunnelClient.app.tar.gz` vs
2. `/TunnelClient.app-1.0.0.tar.gz` and `/TunnelClient.app-somecommitsha.tar.gz`

`oauth-tunnel-client` currently does the 2
I like 1 in case we add docs or change-logs for different versions, we could easily add it to the folder which would be "cleaner".

Reply:
[This one](https://github.com/Shopify/homebrew-shopify/blob/03424de53d5bcc12a7e00c29a3c0e9d90af80a07/oauth-tunnel-client.rb) uses a SHA in the file name. Maybe we should just follow suit?